tools/maven/suppressions.xml

```diff
@@ -35,6 +35,7 @@ under the License.
 		<suppress files="DependencyTree.java" checks="IllegalImport"/>
 
 		<suppress files="FlinkSqlParserImplTest.java" checks="FileLength"/>
+	    <suppress files="SqlDdlToOperationConverterTest.java" checks="FileLength"/>
```

Comment:
can we not split this file into some 2 or more logical groupings rather than suppressing the file length check?

Reply:
Yeah, we can do that. I moved the CTAS tests to another file `SqlCTASNodeToOperationTest`. I also added some tests to `SqlRTASNodeToOperationTest` btw.